lib/rules/no-duplicate-id.js

```diff
@@ -2,7 +2,7 @@ import AstNodeInfo from '../helpers/ast-node-info.js';
 import { match } from '../helpers/node-matcher.js';
 import Rule from './_base.js';
 
-const IGNORE_IDS = new Set(['{{unique-id}}']); // Built-in helper
+const IGNORE_IDS = new Set(['{{unique-id}}', '{{(unique-id)}}']); // Built-in helper, with and without parens
```

Comment:
How about changing this to storing just the helper name? And then the code can check different variations of the usage.
```suggestion
const IGNORED_HELPERS = new Set(['unique-id']); 
```

Reply:
@bmish I gave it a try, and it's overhead for current rule architecture, noted it in TLDR: https://github.com/ember-template-lint/ember-template-lint/pull/2780#issue-1552260436

Reply:
Wouldn't the only difference for this change be:

```js
isIgnoredId(id) {
    return IGNORED_HELPERS.has('{{'+id+'}}') || IGNORED_HELPERS.has('{{('+id+')}}');
}
```

Reply:
@bmish problem not with `{`, problem with `id` itself, it's resolved from `sourceForNode`, not from `path` and this  is main architectural problem for this rule.